### PR TITLE
Add conda-forge

### PR DESCRIPTION
**Pull request by chrisburr**
#### 1Password Teams URL ####

https://conda-forge.1password.com/

#### Project Name ####

conda-forge

#### Short Description ####

A community led collection of recipes, build infrastructure and distributions for the conda package manager.

#### Number Of Core Contributors ####

Currently 26: https://github.com/conda-forge/conda-forge.github.io/blob/master/src/core.csv

#### Project Website ####

https://conda-forge.org/

#### Repository URL(s) ####

There are around 14,000 to enable the ~3200 maintainers in conda-forge to be able to maintain the packages they individually care about. Most repositories are hosted in a dedicated GitHub organisation: https://github.com/conda-forge

Some notable repositories used by the conda-forge infrastructure itself are:

* Repo for adding making new package "feedstocks": https://github.com/conda-forge/staged-recipes/
* Tools for maintaining "feedstocks": https://github.com/conda-forge/conda-smithy/
* Website and documentation: https://github.com/conda-forge/conda-forge.github.io/

#### Latest Release URL(s) ####

There aren't conda-forge "releases" as the main function of conda-forge is to provide binary packages. The binaries are uploaded to several places, most notably:

* https://anaconda.org/conda-forge/
* https://github.com/conda-forge/releases/ (the GitHub web interface is unable to load this repository due to its size but the API works)

#### License Type ####

Packages are distributed under the licence terms of the original software however all conda-forge recipes and infrastructure is under BSD-3-Clause.

#### License URL ####

https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/feedstock_content/LICENSE.txt

#### Name ####

Chris Burr

#### Email ####

christopher.burr@cern.ch

#### Project Role ####

A member of the "core" team which is the governing body over the entire conda-forge organization. See [here](https://conda-forge.org/docs/orga/governance.html) for details.

#### Profile/Website ####

https://github.com/chrisburr/

### CODE REVIEW

**Comment by oliverdunk** [comment]
Hey! 👋 Thanks for your application!

> A is community led collection of recipes, build infrastructure and distributions for the conda package manager.

I think the "is" there is unneeded? Did you want to drop it before I review and merge?

File: README.md [hunk 1]
```diff
@@ -1226,3 +1226,7 @@ https://fd.io
 ### ScanAPI
 ScanAPI is a testing framework that provides automated integration testing and live documentation for any REST API. Given an API specification, written in YAML/JSON format, the library hits the specified endpoints, runs the test cases, and generates a detailed report of this execution - that can be also used as the API documentation itself.
 http://scanapi.dev
+
+### Conda-forge
+A is community led collection of recipes, build infrastructure and distributions for the conda package manager.
```
Comment (beckermr) [inline README.md:1231]:
```suggestion
A community led collection of recipes, build infrastructure and distributions for the conda package manager.
```

**Comment by beckermr** [comment]
I am also on conda-forge. Feel free to remove it and merge!

**Comment by oliverdunk** [comment]
Thanks for your application, @chrisburr! I've gone ahead and applied the promotion to your account.

If you haven't done so already, I would definitely recommend implementing a recovery plan for your team in case access for a particular contributor is ever lost: https://support.1password.com/team-recovery-plan/. You are also welcome to add additional core contributors as you see fit.

Finally, do reach out to us in just under two year's time, as explained in the README. That way, we can look at continuing our support.

**Pull request merged**